pcsx2/gui/AppMain.cpp

```diff
@@ -547,18 +570,35 @@ void Pcsx2App::LogicalVsync()
 	FpsManager.DoFrame();
 	
 	if (EmuConfig.Gamefixes.FMVinSoftwareHack) {
-		if (EnableFMV == 1) {
+		if (EnableFMV) {
 			Console.Warning("FMV on");
 			DoFmvSwitch();
-			EnableFMV = 0;
+			EnableFMV = false;
 		}
 
 		if (FMVstarted){
 			int diff = cpuRegs.cycle - eecount_on_last_vdec;
 			if (diff > 60000000 ) {
```

Comment:
It sounds a lot like the ntsc base frequency multiplied by number of cpu ticks within a second. So I guess it means one frame after the fmv ends... but I don't know. Sounds awful.